virl/files/vinstall.py

```diff
@@ -259,7 +259,10 @@
 keystone_auth_urlv2 = 'http://localhost:5000/v2.0'
 keystone_auth_urlv3 = 'http://localhost:5000/v3'
 #here you can choose v2 or v3 api version
-keystone_auth_url = keystone_auth_urlv3
+# can be chosen by looking at kilo variable too
```

Comment:
I don't see a reason to maintain two copies of vinstall, but reverting to hardcoded url now.